<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunSqlDialog</class>
 <widget class="QDialog" name="RunSqlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="SelectTerminalsForm" name="widget" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonRun">
        <property name="text">
         <string>Выполнить</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/Images/runsql.png</normaloff>:/Images/runsql.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEditSql"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="ViewProgressForm" name="widgetProgress" native="true"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectTerminalsForm</class>
   <extends>QWidget</extends>
   <header location="global">SelectTerminalsForm/selectterminalsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewProgressForm</class>
   <extends>QWidget</extends>
   <header location="global">ViewProgressForm/viewprogressform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
